tests/unit/db/account_tests.py

```diff
@@ -232,14 +234,14 @@ def test_delete_remote_db_via_delitem(self):
         dbname = self.dbname()
         try:
             self.client.connect()
-            db = self.client.create_database(dbname)
+            _ = self.client.create_database(dbname)
             self.assertIsNotNone(self.client.get(dbname))
             self.client.__delitem__(dbname, remote=True)
             # Removed from local cache
             self.assertIsNone(self.client.get(dbname))
             # Database removed remotely as well
             try:
-                db = self.client[dbname]
+                _ = self.client[dbname]
```

Comment:
ah, remove complaints about unused variables to declutter pycharm warnings.  I can revert or simply remove the assignment altogether, let me know.